examples/demo-rollup/benches/rollup_bench.rs

```diff
@@ -0,0 +1,110 @@
+use anyhow::Context;
+use demo_stf::app::NativeAppRunner;
+use demo_stf::runner_config::from_toml_path;
+use std::env;
+use std::sync::Arc;
+
+use sov_demo_rollup::config::RollupConfig;
+use sov_demo_rollup::rng_xfers::RngDaService;
+
+use risc0_adapter::host::Risc0Verifier;
+use sov_db::ledger_db::{LedgerDB, SlotCommit};
+use sov_rollup_interface::mocks::{TestBlock, TestBlockHeader, TestHash};
+use sov_rollup_interface::services::stf_runner::StateTransitionRunner;
+
+use const_rollup_config::SEQUENCER_DA_ADDRESS;
+use demo_stf::genesis_config::create_demo_genesis_config;
+use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
+use sov_rollup_interface::services::da::DaService;
+use sov_rollup_interface::stf::StateTransitionFunction;
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::time::Duration;
+
+mod test_helper;
+
+use test_helper::remove_dir_if_exists;
+
+fn rollup_bench(_bench: &mut Criterion) {
+    let start_height: u64 = 0u64;
+    let mut end_height: u64 = 100u64;
+    if let Ok(val) = env::var("BLOCKS") {
+        end_height = val.parse().unwrap();
+    }
+
+    let mut c = Criterion::default()
+        .sample_size(10)
+        .measurement_time(Duration::from_secs(20));
+    let rollup_config_path = "benches/rollup_config.toml".to_string();
+    let rollup_config: RollupConfig = from_toml_path(&rollup_config_path)
+        .context("Failed to read rollup configuration")
+        .unwrap();
+
+    remove_dir_if_exists(&rollup_config.runner.storage.path).unwrap();
```

Comment:
Just a suggestion, that we can use temp folder here and completely ignore this parameter from config.